content/en/docs/contribution-guidelines.md

```diff
@@ -17,12 +17,12 @@ main website.
 The per-language API, SDK, and "Getting Started" documentation is hosted in each
 language's GitHub repository. For more information,
 [see the Mirrored Documentation section](https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md#mirrored-documentation)
-of the website repository's CONTRIBUTING.md file.
+of the website repository's [CONTRIBUTING.md](https://github.com/open-telemetry/opentelemetry.io/blob/main/CONTRIBUTING.md) file.
```

Comment:
Ditto.

Reply:
a relative link would not work here, since this page is published under https://opentelemetry.io/docs/contribution-guidelines/ and the CONTRIBUTING.md is not published.

Reply:
TIL, thanks!

Reply:
I have the same opinion as @svrnm